apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/config/PortalConfig.java

```diff
@@ -173,6 +173,14 @@ public int connectionTimeToLive() {
     return getIntProperty("api.connectionTimeToLive", -1);
   }
 
+  public int connectPoolMaxTotal() {
+    return getIntProperty("api.pool.max.total", 20);
```

Comment:
What is the default configuration when no connection manager is specified?

Reply:
The default configuration of the connection pool is that the maximum number of connections is 20 and the number of connections per route is 2. This is consistent with the default value of this configuration.